core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -360,6 +361,11 @@ public void setCompressor(Compressor compressor) {}
 
       @Override
       public void setDecompressor(Decompressor decompressor) {}
+
+      // TODO(lukasz) should we return something here?
```

Comment:
An attribute that flags this as "in-process" might be nice. (I think we have code somewhere in our services where they need to distinguish between an in-process request and one from a network client.)